src/app/interfaces/issue/IssueParent.ts

```diff
@@ -0,0 +1,4 @@
+export interface IssueParent {
+  id?: string;
+  name: string;
```

Comment:
Bei GET parent kriegen wir Vor- und Nachname.

```js
{
    "id": "string",
    "firstname": "string",
    "lastname": "string"
}
```

Reply:
Hatte ich gesehen. Da habe ich aber in #30 schon angemerkt, dass ich das nicht ganz verstehe, da ich dachte auch dort die Issue Informationen wie bei Predecessor und Succedssor zu bekommen. Oder für was steht parent?

Reply:
Äh, du hast recht, da macht Vorname nicht mal Sinn 😂 

@Yochyo yoyo schau dir das auch mal an (sorry 4 spam lol)